src/runtime/GlobalObject.cpp

```diff
@@ -1320,5 +1320,16 @@ const Vector<String*, GCUtil::gc_malloc_allocator<String*>>& GlobalObject::intlN
     }
     return m_intlNumberFormatAvailableLocales;
 }
+
+const Vector<String*, GCUtil::gc_malloc_allocator<String*>>& GlobalObject::caseMappingAvailableLocales()
+{
+    if (m_caseMappingAvailableLocales.size() == 0) {
```

Comment:
How about to use just char[][] for this?

Reply:
Unfortunately, `m_caseMappingAvailableLocales` is used for `bestAvailableLocale` method which has fixed argument type of Vector. So I have to follow it.

Reply:
Oh I see!